interface-elements-for-web/articles/gantt/data-editing/modify-dependencies.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Add and Delete Dependencies
+author: Anna Kondratova
+---
+# Add and Delete Dependencies
+
+## Add Dependency
+
+A dependency type specifies which edges (right or left) of the preceding and succeeding tasks serve as the start and end points for the dependency. 
+
+Hold on the preceding task's required edge (marked with a circle) and drag the line to the required edge of the succeeding task. 
```

Comment:
By "drag the line" do you mean "drag the connector"? If so, could we say:

Move the mouse pointer above the preceding task's edge (marked with a circle) and drag the connector to the edge of the next task.

Reply:
Yes, you are right. Applied.